setup/ReShade Setup.csproj

```diff
@@ -37,6 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.WindowsAPICodePack, Version=1.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Yep, I understood what you were suggesting. Also, I totally understand the desire to keep everything a single exe, which is what the additional nuget dependency in the second commit is for :) It handles merging the dependent DLLs into the target executable automatically as part of the build.